pkg/kubeseal-convert/handlers/kubesealconvert/kubeseal.go

```diff
@@ -30,7 +40,7 @@ func Kubeseal(secret string) {
 		Stdout: os.Stdout,
 		Stderr: os.Stdout,
 	}
-
+	var cmdBuffer bytes.Buffer
```

Comment:
moved it inside the function just so we wont have race conditions if called concurrently

Reply:
Nice catch, thanks